### PR TITLE
Remove Beta Code

### PR DESCRIPTION
**Pull request by jcreamer898**
Removing all code referencing split-12-destinations-next.
The global header should always come from rizzo-next now.
Should help resolve lonelyplanet/dispatch#558.

### CODE REVIEW

File: app/views/layouts/custom/_post_header.html.haml [hunk 1]
```diff
@@ -1,13 +1,9 @@
 - if include_js
   %script
-    window.lp.isNewNav = "#{show_new_header?}";
+    window.lp.isNewNav = true;
```
Comment (robuye) [inline app/views/layouts/custom/_post_header.html.haml]: do we still need this?

**Comment by robuye** [comment]
can you also revert this commit too? https://github.com/lonelyplanet/rizzo/commit/a43bc2b8ea60c8253884519910cdb1befd7be13f

File: app/views/layouts/custom/_post_header.html.haml (hunk 1 above)
Comment (WunderBart) [inline app/views/layouts/custom/_post_header.html.haml]: :fire: it :]

File: features/step_definitions/custom_layout_steps.rb [hunk 2]
```diff
@@ -14,9 +14,9 @@
   page.should have_xpath("//link[contains(@href, '/assets/core_fixed_width')]")
 end
 
-Then(/^primary nav should not be present$/) do
-  find('#js-nav--primary').all("*").length.should == 0
-end
+# Then(/^primary nav should not be present$/) do
```
Comment (robuye) [inline features/step_definitions/custom_layout_steps.rb]: ^_^ delete it if not used or needed?

**Comment by WunderBart** [comment]
Looks like something's not right (on my local machine™)
screenshot 2016-03-17 18 10 35

**Comment by WunderBart** [comment]
If you pull master build should be fine. Grunt was set to fetch “latest” version, so it bumped itself to >v1 and caused peer deps errors. It was fixed here: https://github.com/lonelyplanet/rizzo/pull/1505

File: Gemfile.lock [hunk 3]
```diff
@@ -232,6 +232,3 @@ DEPENDENCIES
   typhoeus
   uglifier
   unicorn
-
-BUNDLED WITH
-   1.11.2
```
Comment (WunderBart) [inline Gemfile.lock]: Not sure if this is intended?

File: app/assets/stylesheets/core/layouts/modern/_primary_nav_new.sass [hunk 4]
```diff
@@ -652,32 +682,14 @@ $lp-notification-badge-size: 17px
   [class*=" icon-"]
     @extend %lp-icon-font
 
-  .icon-close-small:before
-    content: "\e605"
+  // .icon-close-small:before
+  //   content: "\e605"
```
Comment (WunderBart) [inline app/assets/stylesheets/core/layouts/modern/_primary_nav_new.sass]: Something important here?

**Comment by WunderBart** [comment]
Works on my machine™ 🎉

File: Gemfile.lock (hunk 3 above)
Comment (robuye) [inline Gemfile.lock]: @jcreamer898 please do `gem update bundler` :)

Comment (jcreamer898) [inline Gemfile.lock]: I swear it will be done.

**Comment by WunderBart** [comment]
There's just one tiny comment left, then GOFORIT 🏃 💥

**Comment by jcreamer898** [comment]
@WunderBart the only thing I'm worried about is Shop.

**Comment by WunderBart** [comment]
I think it will be ok, but also think it won't do any harm if we ask @gnisanci 😉

**Comment by WunderBart** [comment]
omg-gif_2 1

**Comment by jcreamer898** [comment]
hahahahaha

**Pull request merged**